data/speech.bgt

```diff
@@ -0,0 +1,199 @@
+//DFSpeech
+//By DarkFlier Productions
+//Created on 06-19-2016 through 06-20-2016
+class DFSpeech
+{
+string filename; //The filename of the save file for speech settings.
+int bpos; //The buffer position
+string[] b; //The buffer array
+string buffer; //the variable for the filename of output buffer.
+int output; //Internal variable specifying the output mode.
+tts_voice t; //Internal TTS variable
+int mode=0; //Variable for either SAPI or Auto SR output
+int voice;
+int rate;
+int pitch;
+int volume;
+bool say(string text, int interrupt=1, bool wait=false)
+{
+//Speak function
+if (buffer!="")
+{
+if (string_contains(SCRIPT_EXECUTABLE,"Dropbox",1)==-1)
```

Comment:
What is this for?

Reply:
I'm not sure what that's for, I never even noticed it before. I didn't touch the language maniger file more than I needed to.